Sources/Commands/Describe.swift

```diff
@@ -109,7 +109,12 @@ fileprivate struct DescribedPackage: Encodable {
             case .fileSystem(let settings):
                 self = .fileSystem(identity: settings.identity, path: settings.path)
             case .sourceControl(let settings):
-                self = .sourceControl(identity: settings.identity, location: settings.location, requirement: settings.requirement)
+                switch settings.location {
+                case .local(let path):
+                    self = .sourceControl(identity: settings.identity, location: path.pathString, requirement: settings.requirement)
```

Comment:
Should we do a follow-on to separate out the described location similarly?

Reply:
yes we could, less critical at this point imo

Reply:
Yes, absolutely.